lib/galaxy/datatypes/binary.py

```diff
@@ -210,75 +207,19 @@ class Bam(Binary):
     @staticmethod
     def merge(split_files, output_file):
         """
-        Merges BAM files
+        Merges Bam files
```

Comment:
Can we leave this at `BAM files` ? xref https://github.com/galaxyproject/galaxy/pull/5037#discussion_r155796306

Reply:
Done.